Specs/Cataput.txt

```diff
@@ -0,0 +1,19 @@
+A cataput to shoot the balls into the high goal
+
+Needs a mathmatical model of the arm for better control
```

Comment:
Why is this? How we are currently planning on controlling doesn't need a dynamic model. All you need to know is how much string causes an exit angle of `x` and how much spring displacement causes a force `f`.

Reply:
Will change.